setup.sh

```diff
@@ -8,6 +8,7 @@ git clone --recurse-submodules git@github.com:hathitrust/imgsrv
 git clone --recurse-submodules git@github.com:hathitrust/catalog
 git clone --recurse-submodules git@github.com:hathitrust/common
 git clone --recurse-submodules git@github.com:hathitrust/pt
+git clone --recurse-submodules git@github.com:hathitrust/ssd
```

Comment:
Running this script catapulted me into `gh auth` hell. I had hoped installing and using `gh` as my auth helper would be a more "modern" way of doing things -- compared to my previous SSH shenanigans and just checking everything out with my PAT. Alas, `gh` seems to really hate the mixing of Git protocols. I'll spend some more time with this when not crunched for time.

Reply:
Hm, OK. Maybe the best practice would be to use https by default? Not sure.

Reply:
@moseshll Now it asks if you want to use https or ssh before cloning.

Reply:
And I've updated the submodules to use relative paths, so either will work.